tests/githubUtils.test.js

```diff
@@ -289,16 +289,30 @@ describe("Github Utils Tests", () => {
 
   describe("handleSkipOption Tests", () => {
     const mockUpdateLabel = jest.fn();
+    const mockGetContent = jest.fn();
 
+    const octokitMock = {
+      rest: {
+        repos: {
+          getContent: mockGetContent,
+        },
+      },
+    };
```

Comment:
Replace that with `octokitMock = jest.fn()`

Reply:
Good call. Thanks!